modules/service/src/main/scala/lucuma/itc/input/GmosNSpectroscopyInput.scala.~1~

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
```

Comment:
This file may have been accidentally included?

Reply:
Right, I'll delete it